main/chainlink-integration.md

```diff
@@ -0,0 +1,68 @@
+# Chainlink Integration
+
+This document explains how you can consume [Chainlink
```

Comment:
```suggestion
This document explains how to consume [Chainlink
```

"you can consume" reads like Chainlink oracles are a crunchy snack you can chow down on. : -)